### DISCUSSION TITLE
Can someone make me a custom attack like this?

### DISCUSSION THREAD

**Discussion by Chameleon-Coder** [Jcw87/c2-sans-fight#128] [Q&A]
Nevermind.

**Comment by Nightingale08** [comment]
like wat?

**Reply by Chameleon-Coder** [reply]
Dialogue:
(normal) so, uh.......
(glowing eye) why did you do this?
(normal) alright.......
(gaster blasters appear)
(glowing eye) this is my real final attack. ready?

Attacks:

First, the gaster blasters go off. I want them in an X shape.
Then, Sans turns the SOUL blue. A bone stab occurs, from the top, then right, then bottom, then left, then top again.
Next, the SOUL turns red, and the screen is swarmed with blue blasters.
Fourth, he does spinning blasters.
Fifth, he sends bones from the left in the order Orange-Blue-Blue-Orange.
Lastly, he does several orange bone stabs (using the blue SOUL). He then changes the SOUL back to red, and the move repeats.
